src/satipc.c

```diff
@@ -1040,6 +1040,11 @@ void satipc_commit(adapter *ad)
 	if (sip->lap + sip->ldp == 0)
 		if (!sip->force_commit || !ad->tp.freq)
 			return;
+	if (sip->ldp > 0 && (sip->last_cmd == RTSP_TEARDOWN || sip->last_setup < 0))
```

Comment:
This new code fixes the problem _ancient_ pids to remove are requested. In this case (when the session with the target SAT>IP server is empty) the `dpid` list is cleared (these pids are discarded, as they have no sense).